### ISSUE TITLE
`deobfCompile` Does Not Work With Curse Maven

### ISSUE THREAD

**Issue by Desoroxxx** [anatawa12/ForgeGradle-2.3#99]
When using Curse Maven with `deobfCompile` the dependencies just does not appear.

Sample code:

```groovy
repositories {
    maven {
        name 'Curse Maven'
        url "https://cursemaven.com"
        content {
            includeGroup "curse.maven"
        }
    }
}

dependencies {
    deobfCompile 'curse.maven:LLibrary-243298:2504999'
    deobfCompile 'curse.maven:MowziesMobs-250498:3048685'

    deobfCompile 'curse.maven:ElenaiDodge2-442962:3343308'
}
```

This code will run, it seems to download the dependencies, the task won't fail, the dependencies just won't appear.

- Gradle: 7.6.1
- Forge Gradle: 2.3-1.0.7
- Java: jdk8u372-b07

Hope this can be fixed!

**Comment by ACGaming** [comment]
This may specifically target more recent Gradle versions, as this works just fine under Gradle 4.

**Comment by anatawa12** [comment]
Can you show me the output of dependencies task?

**Comment by Desoroxxx** [comment]
> Can you show me the output of dependencies task?

https://mclo.gs/54ZPBe1

With `--warning-mode all`:

https://mclo.gs/a0s19Rc

**Comment by Desoroxxx** [comment]
> This may specifically target more recent Gradle versions, as this works just fine under Gradle 4.

Indeed, that fixes it will work as a workaround for now, I hope this can be fixed soon

**Comment by anatawa12** [comment]
Thank you for providing log. Did you run setup task after adding `deobfCompile`?

**Comment by Desoroxxx** [comment]
> Thank you for providing log. Did you run setup task after adding `deobfCompile`?

If you mean `setupDecompWorkspace` yes

**Comment by anatawa12** [comment]
Thank you for information. I found the reason of this bug. Since Gradle 5.0, Gradle requires pom for maven repository by default but the generated repository for deobfCompile doesn't provide pom. That's why deobfuscated dependencies does not apper. I'm fixing now.

**Comment by anatawa12** [comment]
I've released & published ForgeGradle 2.3-1.0.8, which includes this fix.
